src/SOS/Strike/ExpressionNode.cpp

```diff
@@ -50,10 +50,10 @@ HRESULT ExpressionNode::CreateExpressionNode(__in_z WCHAR* pExpression, Expressi
     *ppExpressionNode = NULL;
 
     HRESULT Status = g_pRuntime->GetCorDebugInterface(&s_pCorDebugProcess);
-    if (FAILED(Status)) 
-    {
+    if (FAILED(Status)) {
```

Comment:
Nit style for the brace

Reply:
I know it isn't exactly our convention but when there is one line in the braces I break the rules instead of not putting braces at all (I really hate that).